source/Symbol/SwiftASTContext.cpp

```diff
@@ -6049,11 +6049,28 @@ SwiftASTContext::GetByteStride(lldb::opaque_compiler_type_t type,
                                ExecutionContextScope *exe_scope) {
   if (!type)
     return {};
+  swift::CanType swift_can_type(GetCanonicalSwiftType(type));
```

Comment:
Can you factor out the code form GetBitSize into a helper function and use it here, too?

Reply:
It's not entirely obvious how to do it. I'll follow-up offline.

Reply:
I will give it a try!